backend/frontend/templates/frontend/user/board/page_card_details.html

```diff
@@ -0,0 +1,117 @@
+{% extends "frontend/user/base.html" %}
+
+{% block main %}
+
+    {% if card.project_submission_type_nice == "link"%}
```

Comment:
If we totally delete the card then the project will still exist. We need to access projects that have no card associated with them.  

The word `card` should not be here. It should just be something like `if project.submission_type == link`